tools/generator/src/Generator+CreateFilesAndGroups.swift

```diff
@@ -286,6 +286,7 @@ extension Generator {
         var allInputPaths = extraFiles
         for target in targets.values {
             allInputPaths.formUnion(target.inputs.all)
+            allInputPaths.formUnion(target.linkerInputs.nonGenerated)
```

Comment:
We use `.nonGenerated` instead of `.all` because generated files will be collected via product outputs, or `extraFiles`.

Reply:
This seems like a salient point. Should we add the above as a code comment?